e2e/test/scenarios/visualizations-charts/maps.cy.spec.js

```diff
@@ -224,4 +224,42 @@ describe("scenarios > visualizations > maps", () => {
       "true",
     );
   });
+
+  it("should apply brush filters by dragging map", () => {
+    visitQuestionAdhoc({
+      dataset_query: {
+        type: "query",
+        database: SAMPLE_DB_ID,
+        query: {
+          "source-table": PEOPLE_ID,
+        },
+      },
+      display: "map",
+      visualization_settings: {
+        "map.region": "us_states",
+        "map.type": "pin",
+        "map.latitude_column": "LATITUDE",
+        "map.longitude_column": "LONGITUDE",
+      },
+    });
+
+    cy.get(".CardVisualization").realHover();
```

Comment:
button "Draw box to filter" appears only on hover, that's the only way I got it working